pkg/util/trace/impl/motrace/report_log.go

```diff
@@ -49,9 +49,18 @@ func (m *MOZapLog) GetName() string {
 	return logView.OriginTable.GetName()
 }
 
+// deltaContentLength approximate value that may gen as table record
+// timestamp: 26
+// level: 5
+// itemName: 8
+// nodeInfo: 40 /*36+4*/
+// spanInfo: 36+16
+const deltaContentLength = int64(26 + 5 + 8 + 40 + 36 + 16)
+
 // Size 计算近似值
 func (m *MOZapLog) Size() int64 {
-	return int64(unsafe.Sizeof(m) + unsafe.Sizeof(len(m.LoggerName)+len(m.Caller)+len(m.Message)+len(m.Extra)))
+	return int64(unsafe.Sizeof(m)) + int64(len(m.LoggerName)+len(m.Caller)+len(m.Message)+len(m.Extra)+len(m.Stack)) +
```

Comment:
@gavinyue ptal, calcualte size of `MOZapLog` with those column may add while output to file.